app/controllers/users_controller.rb

```diff
@@ -1,4 +1,17 @@
 class UsersController < ApplicationController
+  before_action :find_user, only: %i(show edit update correct_user)
```

Comment:
sử dụng except that only được k em

Reply:
Dạ được. Dùng exept thì sẽ thay bằng các method còn lại không có trong chuỗi symbol trên.